rules/combo/backdoor/php.yara

```diff
@@ -277,3 +277,20 @@ rule php_str_replace_obfuscation : critical {
 	condition:
 		filesize < 65535 and 2 of ($f*) and any of ($i*) and 2 of ($o*)
 }
+
+rule php_cgi_argument_injection : critical {
```

Comment:
Technically this isn't a "combo" rule to match a backdoor.

What do you think about a new directory for these kinds of rules, for example: `exploits/php`?

Reply:
I think that makes sense -- TBH the categorizations are still something I'm getting up to speed on 😅

Reply:
Moved in [`8eb5607` (#266)](https://github.com/chainguard-dev/bincapz/pull/266/commits/8eb56076badf207af55fabb41c89a4377dc438e6).